### PR TITLE
Add glossary

### PR DESCRIPTION
**Pull request by el-mapache**
Adds glossary to top navigation area.

* Still need to add tags to each term in the actual markdown
* Glossary code had to be manually imported from a build I made in the main federalist repo. Probably worth our time to submit a PR to the glossary repo with code to create a build file we can copy.

### CODE REVIEW

**Comment by wslack** [comment]
Mobile menu appears broken.
Broken image in the glossary window.

**Comment by el-mapache** [comment]
This should be good to go! The code isn't the prettiest but it will hide the glossary while the content audit is underway.

File: _data/flags.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+glossary: false
```
Comment (adborden) [inline _data/flags.yml]: Why the need to turn the glossary on/off?

Comment (wslack) [reply _data/flags.yml]: @adborden finishing the content will happen in the next month, but we otherwise had no way to put in any of the code.

**Pull request merged**